File: swagger/models/get_active_client_memberships_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .client_membership import ClientMembership

class GetActiveClientMembershipsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		client_memberships
		Details about the requested memberships.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ClientMemberships': ['client_memberships', ClientMembership],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'client_memberships': 'ClientMemberships',
		}

	pagination_response = None
	client_memberships = None

File: swagger/Models/GetActiveClientMembershipsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetActiveClientMembershipsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Details about the requested memberships.
	 * 
	 * @var ClientMembership[] 
	 */
	public $clientMemberships;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'ClientMemberships' => ['clientMemberships', 'array', ClientMembership::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'clientMemberships' => ['ClientMemberships', 'array', ClientMembership::class],
			];
	}
}
